CorePreferences/src/au/gov/asd/tac/constellation/preferences/GraphOptionsPanelController.java

```diff
@@ -0,0 +1,119 @@
+/*

```

Comment:
Please add the copyright header to this file too. If you haven't done so already have a look at https://github.com/constellation-app/constellation/blob/master/CONTRIBUTING.md#pull-requests

Reply:
See latest commit "Resolve PR changes" where the license info is added